composer.json

```diff
@@ -23,15 +23,15 @@
   },
   "require": {
     "php": "^7.3",
-    "phpstan/phpstan": "^0.12.7",
```

Comment:
Since there are no changes in the code itself, it should be possible to specify both versions of PHPStan. 
If there are some trouble with symplify/easy-coding-standard feel free to get rid of it, no need for development tool in a library that's not being developed.